conf/dokuwiki.php

```diff
@@ -1,188 +0,0 @@
-<?php
-/**
- * This is DokuWiki's Main Configuration file
- *
- * All the default values are kept here, you should not modify it but use
- * a local.php file instead to override the settings from here.
```

Comment:
Good point, also it was written just here in front of us all the time...

Reply:
Once a modification is done in the admin area, the local file gets rewritten with the header `This is DokuWiki's Main Configuration file`, weird.
Customizations are conserved